routes/user.routes.js

```diff
@@ -1,6 +1,6 @@
-const router = require('express').Router();
+const usersRouter = require('express').Router();
 const { addUser } = require('../controllers/user.controller');
 
-router.post('/new', addUser);
+usersRouter.post('/new', addUser);
```

Comment:
Suggestion: following REST pattern, it should be interesting to remove `new` subpath, as far as u are using `POST` action